starters/serverless-framework-sqs-dynamodb/README.md

```diff
@@ -0,0 +1,232 @@
+# Serverless Framework, SQS, DynamoDB Kit
+
+## Table of contents
+
+- [Overview](#overview)
+- [Installation](#installation)
+  - [Pre-requisites](#pre-requisites)
+  - [CLI (Recommended)](#cli-recommended)
+  - [Manual](#manual)
+  - [Getting Started](#getting-started)
+- [Available Commands](#available-commands)
+  - [General Commands](#general-commands)
+  - [DynamoDB Commands](#dynamodb-commands)
+  - [Infrastructure Commands](#infrastructure-commands)
+- [DynamoDB](#dynamodb)
+- [Jest](#jest)
+- [Deployment](#deployment)
+
+## Overview
+
+A starter kit utilizing the Serverless Framework in conjunction with AWS DynamoDB & SQS. This kit aims to establish a baseline Serverless Framework project with reasonable configurations that is as-is deployable to AWS. It also provides a fully offline development mode for rapid iteration.
+
+## Installation
+
+### Pre-requisites
+
+- Node.js v16
+- npm or yarn or pnpm
+- Docker & docker-compose
+
+### CLI (Recommended)
+
+1. Init the project
+
+```shell
+npm create @this-dot/starter -- --kit serverless-framework-sqs-dynamodb
+```
+
+or
+
+```shell
+yarn create @this-dot/starter -- --kit serverless-framework-sqs-dynamodb
+```
+
+2. Follow the prompt to name your new project.
+3. `cd` into your project directory.
+
+### Manual
+
+1. Clone the starter.dev repo
+
+```shell
+git clone https://github.com/thisdot/starter.dev.git
+```
+
+2. Copy and rename the `starters/serverless-framework-sqs-dynamodb` directory to the name of your new project.
+3. `cd` into your project directory
+
+### Getting Started
+
+This README uses `yarn` for commands. If you're using `npm` or `pnpm`, utilize the equivalent version of the commands.
+
+1. Create a `.env` file:
```

Comment:
Technically speaking - no. This kit works FULLY offline and does not require AWS at all. That being said, for deployment and other things, it might be nice later. Will add notes.